documentation/src/main/asciidoc/reference/mapper-pojo-standalone.asciidoc

```diff
@@ -46,6 +46,25 @@ include::{sourcedir}/org/hibernate/search/documentation/mapper/pojo/standalone/e
 <4> Build the `SearchMapping`.
 ====
 
+Alternatively a configurer class implementing `StandalonePojoMappingConfigurer` can be provided.
```

Comment:
Shouldn't the example above disappear, if the `addEntities` method is only exposed through the configurer?

Reply:
That builder change actually affected a few more tests/examples from the documentation 🙈